migrator/migrations/n_39_to_n_40_postgres_policies/legacy/remove_test.go

```diff
@@ -0,0 +1,96 @@
+package legacy
+
+import (
+	"context"
+	"encoding/json"
```

Comment:
i think this needs to be jsonpb

Reply:
This is only used to marshal a boolean, not a proto message.